service/cert-signer/cert_signer.go

```diff
@@ -141,40 +141,5 @@ func (cs *certSigner) Issue(config spec.IssueConfig) (spec.IssueResponse, error)
 }
 
 func (cs *certSigner) SignedPath(clusterID string, organizations string) string {
-	return fmt.Sprintf("pki-%s/issue/%s", clusterID, roleName(clusterID, organizations))
-}
-
-func roleName(clusterID string, organizations string) string {
-	if organizations == "" {
-		// If organizations isn't set, use the role that was created when the PKI
-		// for this cluster was first setup.
-		return fmt.Sprintf("role-%s", clusterID)
-	}
-
-	// Compute a url-safe hash of the organizations that stays the same regardless
-	// of the order of the organizations supplied.
-	return fmt.Sprintf("role-org-%s", computeRoleHash(organizations))
-}
-
-// computeRoleHash computes a hash for the role that can issue these organizations.
-// Since we want to reuse roles when possible, we should try to make sure that
-// the same list of organizations returns the same hash (regardless of the order).
-// The reason we don't use just the organizations that the user provided is because
-// that could potentially be a very long list, or otherwise contain characters
-// that are not allowed in URLs.
-func computeRoleHash(organizations string) string {
-	// Sort organizations alphabetically
-	organizationsSlice := strings.Split(organizations, ",")
-	sort.Strings(organizationsSlice)
-	organizations = strings.Join(organizationsSlice, ",")
-
-	h := sha1.New()
```

Comment:
* We started to move everything to go modules and the new architect orb. 
* The new architect orb runs some linters. 
* The linters complain about security issues like using weak crypto libs. Here `sha1`. 
* This changed in `vaultrole` to `sha512` which is why the role names changed. 
* I tested this with `cert-operator` and clusters were just fine. 
* I forgot about `certctl` and we actually should throw it away and only use the new vault roles. 
* I fixed this here now by using the `vaultrole` key function for the actual role name so that we do not have the duplicated implementation anymore.